src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -90,6 +92,8 @@ CRT_RPC_DECLARE(test_swim_status,
 CRT_RPC_DEFINE(test_swim_status,
 	       CRT_ISEQ_TEST_SWIM_STATUS, CRT_OSEQ_TEST_SWIM_STATUS)
 
+int global_client_cb_arg = -1;
```

Comment:
keep this inside of test_t

Reply:
bc783a19b52ef00aa5f17216f0d8c39d9cbf2e8c